lib/plugins/TaskRunner/Script.js

```diff
@@ -36,6 +36,9 @@ module.exports = class Script extends require('./AbstractPlugin') {
       'set -x',
       // Default CWD to $SRC_DIR (create it if it it doesn't exist).
       'mkdir -p $SRC_DIR; cd $SRC_DIR',
+      // add build domain
+      `export BUILD_DOMAIN=${this.container.build.links.build}`,
```

Comment:
BUILD_DOMAIN needs to be setup as an environment variable at container creation: https://github.com/ProboCI/probo/blob/add-generic-lamp-task-plugin/lib/Container.js#L157